src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+    public static void main(String[] args) {
+        Plane firstPlane = new Plane.PlaneBuilder()
+                .setModel("Yak 40")
+                .setCapacity(4)
+                .setType("Passenger")
+                .setYear(2010)
+                .setRange(200)
+                .build();
+
+        System.out.println(firstPlane.toString());
```

Comment:
```suggestion
        System.out.println(firstPlane);
```
you can write like this as well

Reply:
Got it, done